recursion/recursionFibonacci/FibonacciPrinter.go

```diff
@@ -0,0 +1,5 @@
+package FibonacciElements
+
+type FibonacciPrinter interface {
+	recursionFibonacci(n int) int
```

Comment:
use names like computeFibonacci() ... do not include recursion in the name .. as it can be implemented in any way .. name, in general, should not dictate how it to be implemented (like recursion or normal)

Reply:
fixed